src/main/java/by/teachmeskills/service/dto/BookDTOService.java

```diff
@@ -0,0 +1,10 @@
+package by.teachmeskills.service.dto;
+
+import by.teachmeskills.service.BookService;
+
+import java.awt.print.Book;
+import java.util.List;
+
+public interface BookDTOService {
```

Comment:
Service for DTO is not needed - DTO is just for entity to convert and pass DTO to the servlet (not entity)